hijack/contrib/admin/templates/hijack/contrib/admin/button.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n hijack %}
+
+{% if request.user|can_hijack:another_user %}
+<form action="{% url 'hijack:acquire' %}" method="POST">
+  {% csrf_token %}
+  <input type="hidden" name="user_pk" value="{{ another_user.pk }}">
+  <button type="submit" class="button">hijack {{ another_user }}</button>
+  <input type="hidden" name="next" value="{{ request.path }}">
+</form>
+{% endif %}
```

Comment:
```suggestion
{% if request.user|can_hijack:another_user %}
  <form action="{% url 'hijack:acquire' %}" method="POST">
    {% csrf_token %}
    <input type="hidden" name="user_pk" value="{{ another_user.pk }}">
    <button type="submit" class="button">{% trans "Hijack" %}</button>
    <input type="hidden" name="next" value="{{ request.path }}">
  </form>
{% endif %}
```

1. There is not enough indentation.
2. Missed trans
3. A simple "Hijack" in the button will be enough. In the table in the admin panel, it is already clear who this button belongs to.

Reply:
The username is still relevant, when you use the hijack button on a related model. I changed it, so it's only hijack on the user model, but includes the username on any other model.